tdms/include/fieldsample.h

```diff
@@ -8,43 +8,51 @@
 #include "field.h"
 #include "simulation_parameters.h"
 
-class FieldSample{
+class FieldSample {
 
 private:
-  double**** tensor = nullptr;
+  double ****tensor = nullptr;
 
 public:
-  mxArray* mx;       // Matlab array
+  mxArray *mx;// Matlab array
 
-  Vector<int> i;     //< Indices along the x-direction of locations at which to sample the field
-  Vector<int> j;     //< Indices along the y-direction of locations at which to sample the field
-  Vector<int> k;     //< Indices along the z-direction of locations at which to sample the field
-  Vector<double> n;  //< Vector of the moments of the field to sample
+  Vector<int> i;   //< Indices along the x-direction of locations at which to
+                   // sample the field
```

Comment:
Another TODO:
- [ ] Check doxygen correctly renders this (!!!)

Reply:
Doxygen will, however you're documenting a class member so you need the `!<` syntax rather than just `<`:
```
Vector<int> i; //!< Indices along the x-direction of locations at which to 
               //!< sample the field
```

Reply:
Cool. Confirmed. `clang-format` respects the doxygen comments (but the doxygen comment syntax itself needs to be correct).